docs/nconnect-get-started-cli.md

```diff
@@ -0,0 +1,154 @@
+
+- Start nConnect Server:
+Now it's time to start the nConnect server:
+```
+$ ./nConnect -s 
+```
+Great, the nConnect server is started.
+
+# Set Up nConnect Client
+Now we set up the nConnect client on the Windows laptop.
+Just like the server, we download the nConnect source code from github:
+
+- Download nConnect source code
+
+```
+$ git clone https://github.com/nknorg/nconnect
+```
+
+- Build nConnect
+
+```
+$ cd nconnect
+$ go build -o nConnect.exe bin/main.go
+```
+After **go build** it should create **nConnect.exe** executable in the current folder.
+
+- Connecting to the server
+Now let's connect to the nConnect server
+```
+nconnect> ./nConnect -c -a nConnect.l608io.dfc5a05ae4c5f05e4db0f2......522cc90d523a0754fa97
+2023/05/09 11:04:35 get remote server info error: wait for reply timeout. make sure server is online and accepting connections from this client address.
+```
+What!!! You can't connect to the server?
+No, **you can't connect to the server until the server thinks you are authorized.** We need to add the client's address to the server to authorize it. Let's do it.
+
+- Get nConnect client address:
+First we get the client address:
+```
+nconnect> ./nConnect -c --address
+dvbb5p.281ffe4fec4801fc71f188d234c4......927c88b0763ef9e45a9
+```
+Copy this address after **.**, that's **281ffe4fec4801fc71f188d234c4......927c88b0763ef9e45a9**.
+For safety, I use **......** to hide the address details. If you copy or paste it, you should keep the original data.
+
+Okay, now we add this address to nConnect as an authorized address.
+
+- Add client address to server
+Back to the Ubuntu server, we stop nConnect first. In the nConnect folder we can see a file **config.json**, edit it to add the client address:
+```
+"acceptAddrs": ["281ffe4fec4801fc71f188d234c4......927c88b0763ef9e45a9$"],
+ "adminAddrs": ["281ffe4fec4801fc71f188d234c4......927c88b0763ef9e45a9$"]
+```
+Remember to add **$** to the end of the client address, so that any address ending in this string could be accepted by the server.
+For safety, I use **......** to hide address details. if you copy or paste, you should keep the original data.
+
+Ok, this will restart the nConnect server:
+```
+$ ./nConnect -s
+```
+And restart the nConnect client:
+```
+nconnect> ./nConnect -c -a nConnect.l608io.dfc5a05ae4c5f05e4db0f2......522cc90d523a0754fa97
+...
+Client socks proxy listen address: 127.0.0.1:1080
+...
+```
+Good, you can see that the client is listening at the default proxy address: 127.0.0.1:1080. 
+Now client and server have established tunnel. From client Windows laptop, we can connect to Ubuntu PC just like local computer. Let us test it.
+
+### **SSH** Test
+
+From nConnect desktop window, we can see Ubuntu server's local IP address.
+Open Windows CMD window, execute ssh command:
+
+```
+> ssh root@ubuntu-local-ip
+```
+
+Then we see it successful:
+![SSH](./images/nconnect-ssh.png)
```

Comment:
Should it be `./images/nconnect-ssh.png` or `/images/nconnect-ssh.png`?

Reply:
Yes, modified.